get_instagram.py

```diff
@@ -0,0 +1,29 @@
+from InstagramAPI import InstagramAPI
+from key import user_name, password
+
+
+class GetInstagramInfo:
+	def __init__(self, username, password):
+		self.username = username
+		self.password = password
+		self.api = InstagramAPI(username=user_name, password=password)
+		self.api.login()
+		self.user_id = self.api.username_id
+
+	def get_followers(self, user_id=None):
+		"""Returns list of user's followers."""
+		self.user_id = user_id if user_id is not None else self.user_id
```

Comment:
I suggest you remove `self.user_id` field.

You can use it in this way:

    user_id = user_id or self.api.username_id
    followers = self.api.getTotalFollowers(user_id)

Reply:
this is a good idea